v4-client-js/examples/short_term_composite_example.ts

```diff
@@ -0,0 +1,78 @@
+import { Order_TimeInForce } from '@dydxprotocol/v4-proto/src/codegen/dydxprotocol/clob/order';
+
+import { BECH32_PREFIX } from '../src';
+import { CompositeClient } from '../src/clients/composite-client';
+import {
+  Network, OrderExecution, OrderSide,
+} from '../src/clients/constants';
+import LocalWallet from '../src/clients/modules/local-wallet';
+import { Subaccount } from '../src/clients/subaccount';
+import { randomInt } from '../src/lib/utils';
+import { DYDX_TEST_MNEMONIC } from './constants';
+import ordersParams from './human_readable_short_term_orders.json';
+
+async function sleep(ms: number): Promise<void> {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
+
+async function test(): Promise<void> {
+  const wallet = await LocalWallet.fromMnemonic(DYDX_TEST_MNEMONIC, BECH32_PREFIX);
+  console.log(wallet);
+  const network = Network.staging();
+  const client = await CompositeClient.connect(network);
+  console.log('**Client**');
+  console.log(client);
+  const subaccount = new Subaccount(wallet, 0);
+  for (const orderParams of ordersParams) {
+    try {
+      const side = OrderSide[orderParams.side as keyof typeof OrderSide];
+      const price = orderParams.price ?? 1350;
+
+      const currentBlock = await client.validatorClient.get.latestBlockHeight();
+      const nextValidBlockHeight = currentBlock + 1;
+      // Note, you can change this to any number between `next_valid_block_height`
+      // to `next_valid_block_height + SHORT_BLOCK_WINDOW`
+      const goodTilBlock = nextValidBlockHeight + 3;
+
+      const timeInForce = orderExecutionToTimeInForce(orderParams.timeInForce);
+
+      const tx = await client.placeShortTermOrder(
+        subaccount,
+        'ETH-USD',
+        side,
+        price,
+        0.01,
+        randomInt(100_000_000),
+        timeInForce,
+        goodTilBlock,
+        false,
+      );
+      console.log('**Order Tx**');
+      console.log(tx.hash.toString());
+    } catch (error) {
+      console.log(error.message);
+    }
+
+    await sleep(5000);  // wait for placeOrder to complete
+  }
+}
+
+function orderExecutionToTimeInForce(orderExecution: string): Order_TimeInForce {
```

Comment:
Suggestion: do you think this function should be defined within `v4-client-js/src/clients/composite-client.ts` and called within `placeShortTermOrder`?

Reply:
I guess I'm fine with you leaving it here if it's meant to translate from a JSON order

Reply:
this is only meant to translate from the JSON example file, so will keep it in here